doc/en/reference.rst

```diff
@@ -1561,6 +1561,17 @@ passed multiple times. The expected format is ``name=value``. For example::
    See :ref:`change naming conventions` for more detailed examples.
 
 
+.. confval:: required_plugins
+
+   A space separated list of plugins that must be present for pytest to run.
+   If any one of the plugins is not found, emit a error.
```

Comment:
```suggestion
   If any one of the plugins is not found, pytest emits an error and exits with code XXX (unless ``strict_config`` is ``false``).
```

The XXX should be filled here ^^^ (which is it?)

Reply:
strict_config has influence on this as we always fail ( see earlier in PR for reasoning )

Reply:
So since `strict_config` is not consulted in this case, will edit the suggestion above to remove mention of it.

Reply:
Sure I can update this to show the exit code but I think that’s a bit redundant and becomes a maintenance headache since we seem to centralize our exit code information here: https://docs.pytest.org/en/stable/usage.html. I think we should keep exit code information there. If we don’t things will eventually diverge and confuse users

Reply:
Thoughts @nicoddemus @RonnyPfannschmidt

Reply:
It actually looks like it exits with exit code 1, which sounds valid to me for the use case

Reply:
I think it is OK to leave out the exact error code here, but I don't have strong feelings either.

Reply:
If we’re all ok with it, I’d like to leave all exit codes in the usage doc I posted above. Otherwise the error code reporting may diverge over time and people will be confused. I also wouldn’t know to change this location easily as maintainer who has never seen this PR

Reply:
Yea makes sense to leave the exit code out. So just a typo "a error" -> "an error"